tests/winstanley/ReferenceSpec.scala

```diff
@@ -0,0 +1,28 @@
+package winstanley
+
+import java.io.{File, FilenameFilter}
+
+import com.intellij.testFramework.fixtures.LightCodeInsightFixtureTestCase
+
+class ReferenceSpec extends LightCodeInsightFixtureTestCase {
+
+  def testPositiveTaskReference(): Unit = referenceTest("positive_task_reference.wdl")
+
+  def testPositiveVariableReference(): Unit = referenceTest("positive_variable_reference.wdl")
+
+  private def referenceTest(path: String): Unit = {
+    myFixture.configureByFile(path)
+    myFixture.getReferenceAtCaretPositionWithAssertion(path)
```

Comment:
The example [here](https://www.jetbrains.org/intellij/sdk/docs/tutorials/writing_tests_for_plugins/reference_test.html) has a quick string-check to make sure the referred to value has the right content. It's not 100% foolproof but I think "check a reference exists, that it references a task, and that the task we jump to has the right name" is a good start:

Not tested, but something like:
```
private def referenceTest(path: String, expectationAtReference: String) = {
  myFixture.configureByFile(path)
  val element = myFixture.getReferenceAtCaretPositionWithAssertion(path)
    .getReferences()[0].asInstanceOf[WdlNamedElement]

  # Check the name
  assertEquals(expectationAtReference, ((SimpleProperty) element.getValue())

  # Check the type:
  assertTrue(expectationAtReference, ((SimpleProperty) element.isInstanceOf[WdlTaskNameDeclaration])
}
```